doc/tools/python_signatures.py

```diff
@@ -0,0 +1,152 @@
+# TODO: clarify when there are several C++ signatures and only one Python one for the same function.
+#       i.e. calcHist() - http://docs.opencv.org/trunk/d6/dc7/group__imgproc__hist.html#ga4b2b5fd75503ff9e6844cc4dcdaed35d
```

Comment:
Use this instead: http://docs.opencv.org/3.2.0/d6/dc7/group__imgproc__hist.html#ga4b2b5fd75503ff9e6844cc4dcdaed35d